### ISSUE TITLE
Implement referencing other datagraph objects within posts via sdr: URIs

### ISSUE THREAD

**Issue by Southclaws** [Southclaws/storyden#226]
Similar to #185 but this requires some backend work as well as frontend work.

Backend: complete SDR extraction by emitting pubsub messages to draw links between related resources. So when a post or node (any written content resource, basically) uses an SDR URI to reference another datagraph item, the relationships are created in the database.

Also gotta think about removals too, so if a mention/reference is removed from a piece of content, the relationships must be removed too.

Frontend: fairly similar to `@mentions` where `/` (probably, like Notion) is used to bring up a selection+search menu for referencing datagraph items via the existing search endpoint.

Basically like GitHub:

![image](https://github.com/user-attachments/assets/21a0abd6-8806-45c6-88af-62191e87d70e)
